riocli/apply/parse.py

```diff
@@ -379,57 +380,57 @@ def _register_object(self, data):
             return
 
     def _load_file_content(self, file_name, is_value=False, is_secret=False):
-        """Load the file content and return the parsed data.
+        """Load and optionally render a template file, then parse as JSON/YAML."""
+        path = Path(file_name)
+        is_template = path.suffix == ".j2"
+
+        # Resolve real extension (e.g., config.yaml.j2 → .yaml)
+        if is_template:
+            ext = Path(path.stem).suffix.lower()  # second-to-last extension
+        else:
+            ext = path.suffix.lower()
```

Comment:
thought: I think we don't need to support the special .j2 extension. In practice nobody is using this extension. All the files are assumed to be jinja2 templates.